bitmark-references/bitmark-node-software/mining.md

```diff
@@ -0,0 +1,123 @@
+# Overview
+
+This paragraph describes mining mechanism, including hashing, communication protocol, and rewards.
+
```

Comment:
For an introduction like this, it's better to state a general overview then to use a plain listing of what you're going to describe. Something like:

"Mining on the Bitmark blockchain requires an understanding of hashing and difficulty, as well as the communication, verification, and reward protocols."

Reply:
updated